SU2_CFD/src/numerics/flamelet/scalar_diffusion.cpp

```diff
@@ -69,7 +69,7 @@ CAvgGrad_transportedScalar::~CAvgGrad_transportedScalar(void) {
 
 CNumerics::ResidualType<> CAvgGrad_transportedScalar::ComputeResidual(const CConfig* config) {
 
- const bool inc_rans = (config->GetKind_Solver() == INC_RANS);
+ const bool inc_rans = (config->GetKind_Solver() == INC_RANS) || (config->GetKind_Solver() == DISC_ADJ_INC_RANS);
```

Comment:
Eventually we might want to find a better way to do this but for now there might be other instances where this DISC_ADJ_INC_RANS has to be added. Maybe there is some stuff for in the table lookup that uses this construct